### ISSUE TITLE
Component metadata: Subseries level

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#216]
EDIT: As discussed during standup, this work might not need to be done explicitly for Subseries if we do the work in #217 and use it for all components. So #217 should be done before determining if any special work is need for Subseries to get the desired output described below.

For a subseries level component:

<img width="666" alt="Screen Shot 2022-11-14 at 2 42 01 PM" src="https://user-images.githubusercontent.com/101482/201772056-d623f434-0796-4335-99f5-b1018a136a27.png">

---

**Remove the "Extent" field** we currently display for subseries. This doesn't have much value because the Contents section on the page clearly shows how many children there are for this subseries.

**A: Scope and contents**
- Display this first if it exists. A Scope and contents note is appropriate for a subseries, which is why I include it here, but I'm not sure we are having these written at the subseries level. If not, just don't display.

**B: Acquisition information**
- As noted in #214, this field has the same value for all items in the collection but still seems worth displaying.

The end result of the above suggestions is likely that the subseries show page will have "Acquisition information" as the only metadata field we display.

**Comment by ggeisler** [comment]
Closing this in favor of the more general approach in #217.
